consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/TransitionUtils.java

```diff
@@ -45,4 +54,30 @@ public SwitchingObject getPreMergeObject() {
   SwitchingObject getPostMergeObject() {
     return postMergeObject;
   }
+
+  public static boolean isTerminalProofOfWorkBlock(
+      final BlockHeader header, final ProtocolContext context) {
+    Optional<Difficulty> currentChainTotalDifficulty =
```

Comment:
I think the name should reflect that it is the difficulty of the parent block. Something like `parentBlockTotalDifficulty`